pages/ownerInformationPage.ts

```diff
@@ -112,4 +123,24 @@ export class OwnerInformationPage extends HelperBase{
         expect(petVisitsList).toHaveLength(visitsCount)
     }
 
+    async validateTopVisitInformationFor(petName: string, visitDate: Date, visitDescription){
+        const petTable = this.page.locator('table app-pet-list').filter({hasText: petName})
+        const visitRow = petTable.locator('app-visit-list').getByRole('row').nth(1)
+        const visitDateString = this.getDateDashFormat(visitDate)
+
+        await expect(visitRow).toContainText(visitDateString)
+        await expect(visitRow).toContainText(visitDescription)
+    }
+
+    async validateVisitDoesntExist(visitDate: Date, visitDescription: string){
```

Comment:
I think this method is missing the pet name, the visits it's related to. By just the visit date it can pick up the visits for the wrong pet, if there are several pets for the owner.